api/dp-openapi.yaml

```diff
@@ -893,6 +1057,40 @@ components:
         state:
           type: string
 
+    proof_page:
+      type: object
+      properties:
+        pagename:
+          type: string
+        image_url:
+          type: string
+        text:
+          type: string
+
```

Comment:
It would perhaps be useful to send also the number of pages the user can save before reaching their daily page limit.